Tests/SwiftDocCTests/Infrastructure/VersionTripletTests.swift

```diff
@@ -12,6 +12,7 @@ import Foundation
 
 import XCTest
 @testable import SwiftDocC
+@testable import SymbolKit
```

Comment:
Does this import need to be `@testable`? Is the `SemanticVersion` initializer not public?

Reply:
Ah - I misunderstood the meaning of `@testable`. I thought that it was just to indicate we were using the library for testing purposes for informational purposes. But if we don't need that, I'll remove it. thanks!